#!/usr/bin/env python
PACKAGE = "sensors_processing"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()


gen.add("double_param1", double_t, 0, "A double parameter 1", 1., 0., 255.)
gen.add("double_param2", double_t, 0, "A double parameter 2", 1., 0., 255.)
gen.add("morphIterations", int_t, 0, "morph iterations", 1, 0, 20)
gen.add("ringWeight", int_t, 0, "ring weight", 0, 0, 20)

exit(gen.generate(PACKAGE, "sensors_processing", "Tutorials"))
